library/ws-widget/collection/src/lib/_common/content-toc/app-toc-about/app-toc-about.component.html

```diff
@@ -38,7 +38,8 @@
                             <div class="flex items-center certificate_overlay">
                                 <div class="flex flex-col text-center gap-1 p-4 text-white mt-4">
                                     <div>{{'apptochome.certificationTakesTime' | translate }}</div>
-                                    <div>{{'apptochome.issuePersistsMailus' | translate }} <a href="mailto:mission.karmayogi@gov.in" class="emailDefault"> mission.karmayogi@gov.in</a>
+                                    <div>{{'apptochome.issuePersistsMailus' | translate }} mission.karmayogi@gov.in
```

Comment:
Looks like closing div missing

Reply:
No vishnu, checked It is there in the line no 43